### PR TITLE
fix for erroneous deactivation of slow-spinning kinematic objects

### PR DESCRIPTION
**Pull request by AndrewMeadows**

### CODE REVIEW

File: libraries/entities/src/SimpleEntitySimulation.cpp [hunk 1]
```diff
@@ -118,7 +118,7 @@ void SimpleEntitySimulation::changeEntityInternal(EntityItemPointer entity) {
 
             if (entity->isMovingRelativeToParent()) {
                 SetOfEntities::iterator itr = _simpleKinematicEntities.find(entity);
-                if (itr != _simpleKinematicEntities.end()) {
+                if (itr == _simpleKinematicEntities.end()) {
```
Comment (AndrewMeadows) [inline libraries/entities/src/SimpleEntitySimulation.cpp]: The two bugs above were causing moving objects to NOT be put on the list of moving objects on the entity-server.

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/11068/HighFidelity-Beta-PR13525-ff6c309.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/11068/HighFidelity-Beta-PR13525-ff6c309.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/11068/HighFidelity-Beta-Interface-PR13525-ff6c309.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/11068/HighFidelity-Beta-PR13525-ff6c309.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/11068/HighFidelity-Beta-Interface-PR13525-ff6c309.exe

**Comment by qlimnavarro** [comment]
Currently Testing

**Comment by qlimnavarro** [comment]
Pass @conklin94122 
Cube spun and continued to spin even when logged out of domain.

**Pull request merged**